components/datetime/src/provider/calendar/skeletons.rs

```diff
@@ -35,13 +35,13 @@ pub struct DateSkeletonPatternsV1<'data>(
 );
 
 /// Marker type for [`DateSkeletonPatternsV1`].
-#[cfg(feature = "experimental")]
+#[cfg(any(feature = "datagen", feature = "experimental"))]
```

Comment:
Nit/Suggestion: if you need the DateSkeletonPatternsV1Marker to be gated on the same features as DateSkeletonPatternsV1 (previously they were different sets of features), please use the normal derive. I changed it to be manually implemented only so that I could have different feature gates.